hypha/apply/funds/views.py

```diff
@@ -911,7 +928,10 @@ def form_valid(self, form):
                     source=self.object,
                     related=revision,
                 )
-
+        
+        # Required for django-file-form: delete temporary files for the new files
+        # uploaded while edit.
+        form.delete_temporary_files()
```

Comment:
Does this delete files from S3? It might be worth checking and clarifying because with the S3 backend the [tmp files on the local filesystem are deleted](https://github.com/mbraak/django-file-form/blob/master/django_file_form/views/tus.py#L50) as soon as they are uploaded to S3 (and those are still considered temporary until the form is submitted).

Reply:
I hope not as this is being executed after form data has been processed.

Temp files has to be deleted after they are saved in db otherwise they come up again while processing edit forms. It's better that we are deleting temp files at django-file-form when setup with s3 backend. I am not locally setup with s3, maybe that is why I was facing this issue.

@frjo when you deploy this to test env can you please verify that the files are not being deleted from S3.

Reply:
So, the UploadedFile instances you can see in the django-admin are temporary and are on S3 as well. 

And looking at the instructions again you're right to call `.delete_temporary_files()`. I think I just forgot to add it but it is there in the readme.

So, you could maybe just **check in the django admin if they are in fact removed after submit** when this is deployed (but they should be).